pages/index.tsx

```diff
@@ -144,77 +274,258 @@ export default function Home() {
+            color="teal"
+            type="secondary"
+          >
+            Join the waitlist
+          </WebsiteButton>
+        </div>
+        <header className={styles.header}>
+          <p className={styles["eyebrow-headline"]}>
+            Accelerate
+            <span className="green-badge">Early Access</span>
           </p>
+          <h2 className={styles.h2}>Speed up your database queries with an automated global cache</h2>
+          <div>
+            <p>
+              This speed test runs a simple count query on a dataset with 500k rows and shows the results with and without the <a href="https://www.prisma.io/data-platform/accelerate">Accelerate</a> cache.
+            </p>
+          </div>
         </header>
-        <section style={{ gridArea: 'action' }}>
-          <Button
-            autoFocus
-            isDisabled={state === 'running'}
-            onPress={runTest}
-            onClick={runTest}
-            variant={state === 'error' ? 'negative' : 'primary'}
-            type="button"
+
+        <div className={styles.testBtn}>
+          <WebsiteButton
+            color="teal"
+            disabled={state === "running"}
+            onClick={() => runTest()}
           >
-            {state === 'idle' && 'Run Accelerate speed test'}
-            {state === 'running' && 'Running Accelerate speed test'}
-            {state === 'complete' && 'Run another test'}
-            {state === 'error' && 'Try Again'}
-          </Button>
-        </section>
-        <section className={`card ${state}`} style={{ gridArea: 'cache' }}>
-          <h2>✅ With Accelerate</h2>
-          <dl>
-            <dd>{num.format((1_000 / cacheLatency) * 60)}</dd>
-            <dt>queries per minute</dt>
-          </dl>
-          <dl>
-            <dd>{ms.format(cacheLatency)}</dd>
-            <dt>latency</dt>
-          </dl>
-          <span className="badge green">Cached query</span>
-          <Code className="code" value={CODE_CACHE} />
-        </section>
-        <section className={`card ${state}`} style={{ gridArea: 'noCache' }}>
-          <h2>❌ Without Accelerate</h2>
-          <dl>
-            <dd>{num.format((1_000 / withoutCacheLatency) * 60)}</dd>
-            <dt>queries per minute</dt>
-          </dl>
-          <dl>
-            <dd>{ms.format(withoutCacheLatency)}</dd>
-            <dt>latency</dt>
-          </dl>
-          <span className="badge gray">Non-cached query</span>
-          <Code className="code" value={CODE_NO_CACHE} />
-        </section>
-        <section className="results">
-          <h2>Speed test history</h2>
+            {state === "idle" && "Run Accelerate speed test"}
+            {state === "running" && "Running Accelerate speed test"}
+            {state === "complete" && "Run another test"}
+            {state === "error" && "Try Again"}
+            <img src="/arrow-down.svg" />
+          </WebsiteButton>
+        </div>
+
+        <div className={styles.pageInfo} id="testArea">
```

Comment:
The value of the id must be unique within the document, I see this `#testArea` is also used in the next `div`.